packages/dashboard/src/dashboard-actions/actions.ts

```diff
@@ -1,22 +1,25 @@
 import { Action } from 'redux';
-import { Position, Anchor } from '../types';
+import { Position, Anchor, Widget, DashboardState } from '../types';
 
 export const MOVE = 'MOVE';
```

Comment:
What if we create an enum of dashboard-actions, and use that instead of string declarations?

Reply:
that'll look so much nicer, just fixed that. thank you!!